services/apps/portal/src/application/application-document/application-document.entity.ts

```diff
@@ -11,8 +11,17 @@ import {
 import { Document } from '../../document/document.entity';
 import { Application } from '../application.entity';
 
-export const DOCUMENT_TYPES = ['certificateOfTitle'] as const;
-export type DOCUMENT_TYPE = 'certificateOfTitle';
+export const DOCUMENT_TYPES = [
```

Comment:
probably need a ticket to move this types to code tables on ALCS

Reply:
Probably a good idea, its not the nicest method currently